packages/amplication-server/src/core/workspace/workspace.service.ts

```diff
@@ -555,25 +555,96 @@ export class WorkspaceService {
 
     for (const workspaceChunk of workspaceChunks) {
       this.logger.info(`chunk number ${index++}`);
-      await this.migrateWorkspaces(workspaceChunk);
+      await this.migrateWorkspaces(workspaceChunk, false);
     }
 
     await this.prisma.$disconnect();
 
     return true;
   }
 
-  async migrateWorkspaces(workspaces: Workspace[]) {
+  async dataMigrateWorkspacesResourcesCustomActionsFix(): Promise<boolean> {
```

Comment:
There is a ton of repetition here (DRY), but since this code is temporary, I do not mind approving it as-is.

Reply:
@mulygottlieb , I Agree, but that was the decision for not touching the existing code that already has been tested.